Clickstream/EventProcessor/Core/EventProcessor.swift

```diff
@@ -33,6 +33,16 @@ final class DefaultEventProcessor: EventProcessor {
     
     func createEvent(event: ClickstreamEvent) {
         self.serialQueue.async { [weak self] in guard let checkedSelf = self else { return }
+            #if EVENT_VISUALIZER_ENABLED
+            /// Sent event data to client with state received
+            /// to check if the delegate is connected, if not no event should be sent to client
+            if let stateViewer = Clickstream._stateViewer {
+                /// creating the EventData object and setting the status to received.
+                let eventsData = EventData(msg: event.message, state: .received)
```

Comment:
what is `received` state mean?

Reply:
Received state depicts the stage when the event has been received by the clickstream sdk.